documentation/IDTA-01005/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: IDTA-01005
 title: DRAFT - Specification Asset Administration Shell - Part 5 AASX Package Format
-version: 'snapshot'
+version: 'test'
```

Comment:
I prefer snapshot

Reply:
I've changed the version name to generate an additional version in antora, so you can see the generated documentation. We will set it back to snapshot after the commit was approved.

Reply:
or "in progress" like discussed today